app/rev2/prelaunch/test_prelaunch.c

```
@@ -155,24 +161,35 @@ void test_preset_cmd_execute() {
 
 void test_prelaunch_terminal() {
 	uint8_t firmware_code = 0x00;
-	FLASH_STATUS* flash_status;
-	HFLASH_BUFFER* flash_handle;
-	uint32_t* flash_address;
-	uint8_t* gps_msg_byte;
-	SENSOR_STATUS* sensor_status;
+	FLASH_STATUS flash_status;
+	HFLASH_BUFFER flash_handle;
+	uint32_t flash_address;
```

Clang (14.0.6)

[warning] variable 'flash_address' is not initialized

(cppcoreguidelines-init-variables)